scripts/zf_mkdist

```diff
@@ -174,6 +175,7 @@ while [ $# -gt 0 ]; do
         --unstripped) stripped=false;;
         --shim) shim=true;;
         --name) shift; name="$1";;
+        --build_opts) shift; build_opts="$1";;
```

Comment:
I feel like these kinds of multi-word command line options typically get separated with a `-` rather than `_` (so `--build-opts`). Very unimportant suggestion in reality, and perhaps it's just a personal preference thing anyway!

Reply:
Thanks, I think a `-` (dash) is indeed more common than `_` (underscore), and I can never get it right the first time!